sigh/lib/sigh/runner.rb

```diff
@@ -176,6 +176,11 @@ def certificates_for_profile_and_platform
           certificates = Spaceship.certificate.development.all
         elsif profile_type == Spaceship.provisioning_profile.InHouse
           certificates = Spaceship.certificate.in_house.all
+        # handles case where the desired certificate type is adhoc but the account is an enterprise account
+        # the apple dev portal api has a weird quirk in it where if you query for distribution certificates
+        # for enterprise accounts, you get nothing back even if they exist.
+        elsif profile_type == Spaceship.provisioning_profile.AdHoc && Spaceship.client.in_house?
```

Comment:
We _may_ need to add an additional check here to see make sure `Spaceship.client` isn't `nil`? I don't think it can but safety first?

I'd rather have an extra check that _may_ not need be needed than a potential crash 🙃

Reply:
Agreed! My ruby fu is a little rusty, would that be:

```ruby
elsif !Spaceship.client.nil? && Spaceship.client.in_house? && Spaceship.provisioning_profile.AdHoc
```

Reply:
@joshdholtz

Reply:
@startupthekid The would personally do...
```rb
elsif profile_type == Spaceship.provisioning_profile.AdHoc && Spaceship.client && Spaceship.client.in_house?
```